config/environments/production.rb

```diff
@@ -26,6 +26,10 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
```

Comment:
Should this be uncommented?

Reply:
No, we don't use the master key . The comment was added in the Rails 5.2 update generator.